nsv13/code/modules/overmap/overmap.dm

```diff
@@ -195,6 +195,7 @@
 	var/reserved_z = 0 //The Z level we were spawned on, and thus inhabit. This can be changed if we "swap" positions with another ship.
 	var/list/occupying_levels = list() //Refs to the z-levels we own for setting parallax and that, or for admins to debug things when EVERYTHING INEVITABLY BREAKS
 	var/torpedo_type = /obj/item/projectile/guided_munition/torpedo
+	var/missile_type = /obj/item/projectile/guided_munition/missile
```

Comment:
Unused variable. Hook into `fire_missile()`.

Reply:
Thank you! I'd wondered what I missed!